### DISCUSSION TITLE
JSON format document for flow import and export?

### DISCUSSION THREAD

**Discussion by mapachekurt** [FlowiseAI/Flowise#3428] [Q&A]
Is there a source document for how the json should be structured for import/export of flows? If so, an LLM could be trained to create flows based on prompting. 

I'm surely not the first person to think this, but I couldn't find any threads with this subject?

**Comment by starfoxbeta** [comment]
I'm looking for similar answers.

**Comment by CoderCaleb** [comment]
bump
